Gaming (#518)
solana-developers/solana-cookbook: docs/gaming/interact-with-tokens.md

---
title: How interact with tokens in programs
head:
  - - meta
    - name: title

>       content: Solana Cookbook | How interact with tokens in programs

Reviewer: ```suggestion
      content: Solana Cookbook | Using tokens in games on Solana
```

  - - meta
    - name: og:title

>       content: Solana Cookbook | How interact with tokens in programs

Reviewer: ```suggestion
      content: Solana Cookbook | Using tokens in games on Solana
```

  - - meta
    - name: description

>       content: A small game that shows how game can interact with the Solana block chain

Reviewer: ```suggestion
      content: Learn how to use tokens in Solana games with an on-chain tutorial
```

  - - meta
    - name: og:description

>       content: A small game that shows how game can interact with the Solana block chain

Reviewer: ```suggestion
      content: Learn how to use tokens in Solana games with an on-chain tutorial
```

  - - meta
    - name: og:image
      content: https://solanacookbook.com/cookbook-sharing-card.png
  - - meta
    - name: og:image:alt
      content: Solana splash card
  - - meta
    - name: twitter:card
      content: summary
  - - meta
    - name: twitter:site
      content: "@solanacookbook"
  - - meta
    - name: twitter:image
      content: "https://solanacookbook.com/cookbook-sharing-card.png"
  - - meta
    - name: robots
      content: index,follow,noodp
  - - meta
    - name: googlebot
      content: index,follow
footer: MIT Licensed
---

> # How interact with tokens in programs

Reviewer: ```suggestion
# Using tokens in games on Solana
```
